cobra/Makefile

```diff
@@ -0,0 +1,23 @@
+XC_OS="linux darwin"
```

Comment:
Why linux and darwin only? Is there any issue with windows?

Reply:
The intention is to provide defaults.  These variables can be overridden when calling `make`.  We do not currently build for Windows, but this is laying the groundwork to build and publish binaries for various operating systems and architectures.

Reply:
Do you mean to provide an example about how users could publish their tools? Or to actually publish cobra cli tool periodically? In the second case, and just because of curiosity, may I know why not https://github.com/goreleaser/goreleaser, which is quite popular?

Reply:
I mean to publish the CLI tool periodically. Honestly was not aware of the tool you have mentioned.  I'll look into it but for now gox works fine as all I want to do is build a few binaries in parallel and stick them in the bin directory.